docs/openapi3.0.yml

```diff
@@ -19,35 +19,56 @@ components:
           type: string
         type: 
           type: string
-          enum: ["user", "pro", "ai"]       
-    Question:
+          enum: ["user", "pro", "ai"]
+
+    UserContent:
       type: object
       properties:
-        id: 
-          type: string
-        title: 
+        id:
           type: string
-        content: 
+        content:
           type: string
-        tags: 
-          type: array
-          items:
-            type: string
-        likes: 
+        likes:
           type: integer
         dislikes:
           type: integer
-        answers:
-          type: integer
         created:
           type: string
           description: The date, the question was created
-        updated: 
+        rating:
```

Comment:
rename so that its clear, that the rating is the rating of the current user

Reply:
Is _optinion_ a valid alternative?

Reply:
currentUserRating?